src/Grpc.AspNetCore.Server/Internal/ObjectMethodExecutor/AwaitableInfo.cs

```diff
@@ -0,0 +1,142 @@
+#region Copyright notice and license
```

Comment:
qq: the original file has a .NET foundation license header:
https://github.com/aspnet/AspNetCore/blob/1aa50faa290ecda304507981cd01ed92651d5e34/src/Shared/ObjectMethodExecutor/AwaitableInfo.cs#L1

I am not expert on licensing but my impression was that we need to keep the original header (and add a gRPC license header on top of that?)?
https://github.com/aspnet/AspNetCore/blob/1aa50faa290ecda304507981cd01ed92651d5e34/src/Shared/ObjectMethodExecutor/AwaitableInfo.cs#L1

Reply:
@jtattermusch in this particular case it's OK because the original author of the code is giving the code to this project (the author is an MSFT person). So, in a sense this isn't taking code from another OSS project, it's the original author contributing the same code twice.

Reply:
sounds good, thanks for explaining.